### DISCUSSION TITLE
Add control media button

### DISCUSSION THREAD

**Discussion by kikiimdev** [namidaco/namida#373] [Ideas]
Add an option to treat media button (like button on headset/buds) as player control for next/prev or fast forward/rewind

![Example option on MX Player](https://github.com/user-attachments/assets/7d1e5a2e-5343-44af-8d04-5e3a1db400e0)

**Comment by MSOB7YY** [comment]
this now at #374

**Comment by MSOB7YY** [comment]
kinda late but this was added back in 4.8.x
latest beta here https://github.com/namidaco/namida-snapshots/releases
